sdk/src/main/java/nl/rabobank/gict/payments_savings/omnikassa_frontend/sdk/Endpoint.java

```diff
@@ -67,8 +91,42 @@ public static Endpoint createInstance(String baseURL, byte[] signingKey, TokenPr
     public static Endpoint createInstance(Environment environment,
                                           String base64encodedSigningKey,
                                           TokenProvider tokenProvider) {
+        return createInstance(environment, base64encodedSigningKey, tokenProvider, null, null);
+    }
+
+    /**
+     * Use this method to retrieve an instance of the Endpont
+     *
+     * @param environment             this is the Url that points to the Rabobank API
+     * @param base64encodedSigningKey this is the key given by the Rabobank to sign all communication
+     * @param tokenProvider           this must be your own implementation of the tokenProvider, see developer-manual
+     * @param userAgent               this is the User-Agent value you want to give your implementation
+     * @return new instance of Endpoint
+     */
+    public static Endpoint createInstance(Environment environment,
```

Comment:
I think providing a constructor with both the userAgent and partner reference (the one below) is enough. This one can be removed